src/patterns/start-pages/index.md.njk

```diff
@@ -1,41 +1,72 @@
 ---
-title: Start pages
-description: A starting point for your digital service on GOV.UK
+title: Start using a service
+description: Create a starting point for your digital service on GOV.UK
 section: Patterns
-theme: Pages
-aliases:
+theme: Help users to…
+aliases: start page, start pages
 backlog_issue_id: 111
 layout: layout-pane.njk
 ---
 
-{% from "_example.njk" import example %}
+GOV.UK services must start on a GOV.UK content page. This links your service to the rest of GOV.UK, and means that users can find it using search or through GOV.UK’s navigation.
 
-Use this pattern to test a start page in your prototype.
+## When to use this pattern
 
-To get a start page for your public beta or live service, you need to [contact the GDS content team](https://www.gov.uk/service-manual/service-assessments/get-your-service-on-govuk) before your alpha assessment.
+Use this pattern to help you prototype a start point for your service, so you can test as much of the user journey as possible.
 
-{{ example({group: "patterns", item: "start-pages", example: "default", html: true, nunjucks: true, open: false}) }}
+The public-facing version of your service will need to start on a GOV.UK content page, published using one of the GOV.UK publishing apps. You’ll usually need to agree content for this GOV.UK content page with the GDS content team. Read [guidance on how to get your service on GOV.UK](https://www.gov.uk/service-manual/service-assessments/get-your-service-on-govuk).
 
-## When to use this pattern
+## How it works
 
-If your service is in public beta or live, you must have a starting point hosted on GOV.UK. This will link your service to the rest of GOV.UK.
+A service start point should:
```

Comment:
```suggestion
A service's start point should:
```
'Start point' or 'starting point'?

Did a quick site-search and 'starting point' looks like [what we've used before](https://www.google.com/search?q=%3Csite%3Adesign-system.service.gov.uk%2F+%22starting+point%22%3E&rlz=1C5CHFA_enGB970GB970&oq=%3Csite%3Adesign-system.service.gov.uk%2F+%22starting+point%22%3E&aqs=chrome..69i57.5666j0j4&sourceid=chrome&ie=UTF-8). Didn't see any results for 'start point'...

Reply:
Looks like the term is only used on this page, I slightly prefer 'start point' to be a bit tidier.